service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/methods/MethodResult.java

```diff
@@ -21,12 +23,27 @@
     /**
      * Payload with the result of the Invoke Method
      */
-    @Getter
-    private final Object payload;
+    private final JsonElement payload;
 
-    public MethodResult(Integer status, Object payload)
+    public MethodResult(Integer status, JsonElement payload)
```

Comment:
Rename this class to DirectMethodResponse for consistency with other direct method classes